### ISSUE TITLE
jsx-indent rule doesn't properly validate ternary (?:) expressions with no parens

### ISSUE THREAD

**Issue by jrajav** [yannickcr/eslint-plugin-react#1315]
Prior related issues:

https://github.com/yannickcr/eslint-plugin-react/issues/540
https://github.com/yannickcr/eslint-plugin-react/issues/625

Patterns:

```js
{ condition
  ? <div>
      <div> something </div>
    </div>
  : <div>
      <div> something </div>
    </div>
}

{ condition
    ? <div>
        <div> something </div>
      </div>
    : <div>
        <div> something </div>
      </div>
}

{
  condition
  ? <div>
      <div> something </div>
    </div>
  : <div>
      <div> something </div>
    </div>
}

{
  condition
    ? <div>
        <div> something </div>
      </div>
    : <div>
        <div> something </div>
      </div>
}
```

**Expected behavior:**
All of these patterns properly validate and flag no errors.

**Actual behavior:**
All of these patterns flag errors for lines following and indented after the ternary result clauses (after ? and : lines).

**Comment by jrajav** [comment]
Note: Also an issue with jsx-indent-props, e.g.:

```js
{
  condition
    ? <El
        prop1={ 1 }
        prop2={ 2 }
      />
    : <OtherEl
        prop1={ 1 }
        prop2={ 2 }
      />
}
```

I could make a separate issue for this if necessary.

**Comment by derekjobst** [comment]
I've also experienced these issues. Any idea what steps we could take to get moving on a fix?

**Comment by ljharb** [comment]
Someone coming forward with a PR would be helpful; however, #1317 may have mitigated this and isn't yet released, afaik.

**Comment by ljharb** [comment]
I've pushed up a [branch](https://github.com/jsx-eslint/eslint-plugin-react/compare/HEAD...ljharb:eslint-plugin-react:issue_1315) with failing tests, in case anyone can pick this up.

**Comment by root0x** [comment]
Does anyone have any updates on this?
